app/experimenter/templates/experiments/detail_base.html

```diff
@@ -31,38 +31,30 @@ <h3>
   {% endif %}
 {% endblock %}
 
-{% block header_links %}
-  {% if experiment.has_external_urls %}
-    <div class="col py-2">
-      {% if experiment.bugzilla_url %}
-        <a class="alert-link mr-4" target="_blank" rel="noreferrer noopener" href="{{ experiment.bugzilla_url }}">
-          <img class="link-icon" src="{% static "imgs/bugzilla.png" %}" alt="bugzilla">
-          Delivery Bugzilla Ticket
-        </a>
-      {% endif %}
-
-      {% if experiment.data_science_bugzilla_url %}
-        <a class="alert-link mr-4" target="_blank" rel="noreferrer noopener" href="{{ experiment.data_science_bugzilla_url }}">
-          <img class="link-icon" src="{% static "imgs/bugzilla.png" %}" alt="bugzilla">
-          Data Science Bugzilla Ticket
-        </a>
-      {% endif %}
-
-      {% if experiment.feature_bugzilla_url %}
-        <a class="alert-link mr-4" target="_blank" rel="noreferrer noopener" href="{{ experiment.feature_bugzilla_url }}">
-          <img class="link-icon" src="{% static "imgs/bugzilla.png" %}" alt="bugzilla">
-          Feature Bugzilla Ticket
-        </a>
-      {% endif %}
-
-      {% if experiment.monitoring_dashboard_url %}
-        <a class="alert-link" target="_blank" rel="noreferrer noopener" href="{{ experiment.monitoring_dashboard_url }}">
-          <span class="fas fa-chart-bar"></span>
-          Live Monitoring Dashboard
-        </a>
-      {% endif %}
+{% block progress_bar %}
+<div class="container">
+  <div class="row my-4">
+    <div class="col-9 h-100">
+      <div class="row whole-bar">
+        {% for status in experiment.progress_bar_highlighted %}
+          {% if status.1 %}
+            <div class="col-2 bar-piece-highlighted">
```

Comment:
You can actually put the if statement right inside the class attribute since the rest of the body of this is the same like

`class="col-2 bar-piece{% if status.1 %}-highlighted{% endif %}"`